src/css/app.css

```diff
@@ -0,0 +1,41 @@
+@font-face {
+    font-display: swap;
+    font-family: 'Source Sans Pro';
+    font-style: normal;
+    font-weight: 300;
+    src: url('../fonts/sourcesanspro-light-webfont.woff2') format('woff2');
```

Comment:
Vite can automatically copy & hash filenames for fonts. It just has to be a relative path from the css file to the font file.